### PR TITLE
Add API for creating job via test-composer

### PR DESCRIPTION
**Pull request by tianfeng92**
Create this API for creating job via test-composer. This is an internal API and a work around for creating job currently.
Usage:
```
./bin/sl createJob '{}'
```

### CODE REVIEW

File: apis/testrunner.json [hunk 1]
```diff
@@ -136,6 +136,41 @@
...
+        ],
+        "externalDocs": {
+          "description": "TBD",
+          "url": "https://wiki.saucelabs.com"
+        },
+        "tags": [
+          "Job"
+        ],
+        "responses": {
+          "200": {
```
Comment (alexplischke) [inline apis/testrunner.json:162]: For a creation endpoint, I think
```suggestion
          "201": {
```
is more appropriate :D

**Review by alexplischke — APPROVED** [review]

**Review by alexplischke — CHANGES_REQUESTED** [review]
Just one comment!

File: apis/testrunner.json (hunk 1 above)
Comment (tianfeng92) [reply apis/testrunner.json]: make sense! updated.

**Review by christian-bromann — APPROVED** [review]
👍

**Review by alexplischke — APPROVED** [review]

**Pull request merged**